helm/forgerock-metrics/README.md

```diff
@@ -23,19 +33,20 @@ Alertmanager configuration: [Config](https://prometheus.io/docs/alerting/configu
 The Prometheus Operator works by watching for ServiceMonitor CRDs (CRDs are Kubernetes Custom Resource Definitions). These are first  
 class Kubernetes types that you can manage with kubectl (kubectl create/delete/patch, etc.).  The ServiceMonitor CRDs define the target to be scraped.
 
-The Prometheus Operator watches for changes to current ServiceMonitors or for new ServiceMonitors and updates the  
-Prometheus configuration with the details defined in the ServiceMonitors automatically.  
+The Prometheus Operator watches for changes to current or new ServiceMonitors and updates the Prometheus configuration with the details  
+defined in the ServiceMonitors automatically.  
 
 No restarting of Prometheus is required.
 
 <br />
 
 # How Grafana works
 
-The Grafana Helm chart is deployed as part of the kube-prometheus chart.  This comes prepackaged with ServiceMonitors and  
-dashboards that monitor various aspects of the GKE cluster including the cluster node resources and Kubernetes objects.
+The Grafana Helm chart is deployed as part of the kube-prometheus chart.  Grafana automatically connects to Prometheus and syncs all  
+the metrics which are visible through Graphs.  
 
-Dashboards for ForgeRock products are imported into Grafana after Grafana has been deployed. 
+Dashboards for ForgeRock products are imported into Grafana post deployment. This is carried out by a Kubernetes job that mounts the dashboard files  
```

Comment:
s/post deployment/post-deployment/
s/This is carried out by a Kubernetes job that mounts/A Kubernetes job mounts/
Period at the end of this sentence, pls